### ISSUE TITLE
Volcano: update legend text and pointer

### ISSUE THREAD

**Issue by asizemore** [VEuPathDB/web-monorepo#558]
**Problem:** 
Text in the legend is unclear. Also the cursor switches to the hand when over the legend which suggests it's interactive but the legend isn't interactive. (from slack)

**Suggested solution:**
Change the legend text to "Up in X" and "Down in X". Also keep the mouse the usual pointer while over the legend to prevent the user from thinking it's interactive.

**Comment by moontrip** [comment]
Hi @asizemore 👋 

FYI, I implemented the pointer at legend long time ago. So, at first, I looked back the reason why a pointer was used in there. After checking them, I recalled that it was introduced because of the tooltip of the legend labels, not indicating interactive behavior. Thus, in short, we cannot remove the pointer in there unless we disable mouse-over tooltip. 

Actually, it is a CSS, `pointer-events`, that enables to select/point out a SVG element so that corresponding tooltip belong to the SVG element. Thus, if we disable the CSS, then the mouse-over tooltip will not work because it cannot find associated SVG element. 

Hope this would be helpful.
